Firestore/core/src/firebase/firestore/model/transform_operations.h

```diff
@@ -93,58 +90,49 @@ class ServerTimestampTransform : public TransformOperation {
     return Type::ServerTimestamp;
   }
 
-// TODO(mikelehen): apply methods require FSTFieldValue which is Obj-C only.
-#if defined(__OBJC__)
-  FSTFieldValue *applyToLocalView(FSTFieldValue *previousValue,
-                                  FIRTimestamp *localWriteTime) const override {
+  FSTFieldValue* ApplyToLocalView(FSTFieldValue* previousValue,
+                                  FIRTimestamp* localWriteTime) const override {
     return [FSTServerTimestampValue
         serverTimestampValueWithLocalWriteTime:localWriteTime
                                  previousValue:previousValue];
   }
 
-  FSTFieldValue *applyToRemoteDocument(
-      FSTFieldValue *previousValue,
-      FSTFieldValue *transformResult) const override {
+  FSTFieldValue* ApplyToRemoteDocument(
+      FSTFieldValue* /* previousValue */,
```

Comment:
Did I miss an unused variable warning somewhere? :-/

Reply:
Yes. Most visible in the CMake build where this is an error. This did not fail previously because it wasn't compiled there until I removed the conditional compilation.